awscli/customizations/s3/utils.py

```diff
@@ -441,6 +441,16 @@ def map_get_object_params(cls, request_params, cli_params):
         cls._set_sse_c_request_params(request_params, cli_params)
         cls._set_request_payer_param(request_params, cli_params)
 
+    @classmethod
```

Comment:
We should probably update the unit tests to add cases for these new methods. There should be a `TestRequestParamsMapperRequestPayer` class that can be updated in `tests/unit/customizations/s3/test_utils.py`